springfox-spring-web/src/main/java/springfox/documentation/spring/web/scanners/ApiListingScanner.java

```diff
@@ -127,7 +126,7 @@ public ApiListingScanner(
 
     List<SecurityReference> securityReferences = new ArrayList<>();
 
-    Map<String, Set<Model>> globalModelMap = new TreeMap<>();
+    Map<String, Set<Model>> globalModelMap = new HashMap<>();
```

Comment:
This was tree map because it preserves the order of the models. Does this need to change to hash map?

Reply:
Iterating over HashMap is faster than iterating TreeMap, and order is not important any more